tests/spec/diagnostics_spec.lua

```diff
@@ -10,6 +10,7 @@ local helpers = require("cspell.helpers")
 local uv = vim.loop
 
 local CSPELL_CONFIG_PATH = uv.fs_realpath("./cspell.json")
+local CSPELL_MERGED_CONFIG_PATH = Path:new("."):joinpath(".tests/cache/nvim/cspell.nvim/cspell.json"):expand()
```

Comment:
The tests pass if we change this path to:

```suggestion
local CSPELL_MERGED_CONFIG_PATH = Path:new(".tests/cache/nvim/cspell.nvim/cspell.json"):expand()
```

Reply:
Fixed in latest change!